### DISCUSSION TITLE
Creating multiple views (Linux support)?

### DISCUSSION THREAD

**Discussion by dylanbmckinney** [unoplatform/uno#5331] [Q&A] [answered]
When creating a UWP application, one may use something like CoreApplication.CreateNewView() to add a view to the application (in a new window). Unfortunately, this part of Windows.ApplicationModel.Core.CoreApplication does not port when I create a build for the .Skia.Gtk project. Is there a known workaround for adding multiple views or supporting multiple monitors across Windows and Linux? I've looked around for an easy solution to this but so far I've come up empty handed.

Bonus points if the solution also works in Raspbian.

I've attached a basic project which includes a button that spawns new views.

The specific build warnings I see are:
Windows.ApplicationModel.Core.CoreApplication.CreateNewView() is not implemented in Uno
Windows.UI.ViewManagement.ApplicationView.Id is not implemented in Uno
Windows.UI.ViewManagement.ApplicationViewSwitcher.TryShowAsStandaloneAsync(int) is not implemented in Uno

In lieu of multiple views, one fullscreen view that spans multiple monitors is also an acceptable workaround for my situation.

[MultipleWindows.zip](https://github.com/unoplatform/uno/files/6051595/MultipleWindows.zip)

**Comment by jeromelaban** [comment, marked as answer]
As of Uno 3.6, this is indeed not yet supported. We're tracking those features in #4309, if you want to follow.
